Plugin.cs

```diff
@@ -42,24 +51,44 @@ private void Awake()
                 Logger.LogError("Disabling VR because assets could not be loaded!");
                 return;
             }
-
+            ConfigBind();
+            UpdateCongiurableValues();
             // TODO: Make this configurable
             var asset = QualitySettings.renderPipeline as HDRenderPipelineAsset;
             var settings = asset.currentPlatformRenderPipelineSettings;
 
-            settings.dynamicResolutionSettings.enabled = true;
-            settings.dynamicResolutionSettings.enableDLSS = false;
-            settings.dynamicResolutionSettings.dynResType = DynamicResolutionType.Hardware;
-            settings.dynamicResolutionSettings.upsampleFilter = DynamicResUpscaleFilter.CatmullRom;
-            settings.dynamicResolutionSettings.minPercentage = 25;
-            settings.dynamicResolutionSettings.maxPercentage = 50;
-
             asset.currentPlatformRenderPipelineSettings = settings;
             
             Logger.LogInfo("Loading VR...");
             InitVRLoader();
         }
 
+        void ConfigBind()
+        {
+            EnableDynamicRes = Config.Bind("Settings", "Enable Dynamic Resolution", true);
+            EnableDLSS = Config.Bind("Settings", "Enable DLSS", false);
+            DynamicResType = Config.Bind("Settings", "Type of Dymaic Resolution", DynamicResolutionType.Hardware);
+            UpscaleFilter = Config.Bind("Settings", "Type of Upscale Filter", DynamicResUpscaleFilter.CatmullRom);
+            DynamicResMinPercent = Config.Bind("Settings", "Minimum Percent for Res", 25f);
+            DynamicResMaxPercent = Config.Bind("Settings", "Max Percent For Res", 50f);
+            //TODO add a way to update the config if they change
+            // there is a Config.SettingChanged event
+            
+        }
+
+        void UpdateCongiurableValues()
+        {
+            var asset = QualitySettings.renderPipeline as HDRenderPipelineAsset;
+            var settings = asset.currentPlatformRenderPipelineSettings;
+
+            settings.dynamicResolutionSettings.enabled = EnableDynamicRes.Value;
+            settings.dynamicResolutionSettings.enableDLSS = EnableDLSS.Value;
+            settings.dynamicResolutionSettings.dynResType = DynamicResType.Value;
+            settings.dynamicResolutionSettings.upsampleFilter = UpscaleFilter.Value;
+            settings.dynamicResolutionSettings.minPercentage = DynamicResMinPercent.Value;
+            settings.dynamicResolutionSettings.maxPercentage = DynamicResMaxPercent.Value;
+        }
```

Comment:
You aren't actually setting anything here, just reading it into a local RenderPipelineSettings object. Mind if I send a commit to address this? I've also got one lined up to subscribe to Config.SettingChanged.

Reply:
That's in that void incase they change the config and it can auto update as they change it